Bottom: 7da2b7ee5b4ad21ebd267a6cfaa60f6ae58e643c
Top:    d89ff5f7873fbe1cf14d93ee24ce3e3fea286bb6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-10-20 16:37:14 +0100

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index dacc45e..daee7bf 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -831,19 +831,20 @@ fill_contour (PLINE *contour)
   if (contour->tristrip_num_vertices == 0)
     return;
 
+#if 0
   hidgl_ensure_vertex_space (&buffer, contour->tristrip_num_vertices);
   memcpy (&buffer.triangle_array[buffer.coord_comp_count],
           contour->tristrip_vertices,
           sizeof (float) * 5 * contour->tristrip_num_vertices);
   buffer.coord_comp_count += 5 * contour->tristrip_num_vertices;
   buffer.vertex_count += contour->tristrip_num_vertices;
-
-#if 0
+#else
   vertex_comp = 0;
   for (i = 0; i < contour->tristrip_num_vertices; i++) {
     int x, y;
     x = contour->tristrip_vertices[vertex_comp++];
     y = contour->tristrip_vertices[vertex_comp++];
+    vertex_comp += 3;
     hidgl_add_vertex_tex (&buffer, x, y, 0.0, 0.0);
   }
 #endif
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 459d561..f803fc3 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -982,9 +982,9 @@ ghid_init_renderer (int *argc, char ***argv, GHidPort *port)
   /* setup GL-context */
   priv->glconfig = gdk_gl_config_new_by_mode (GDK_GL_MODE_RGBA    |
                                               GDK_GL_MODE_STENCIL |
-                                              GDK_GL_MODE_DEPTH  );// |
+                                              GDK_GL_MODE_DEPTH   |
                                            // GDK_GL_MODE_DEPTH   |
-//                                              GDK_GL_MODE_DOUBLE);
+                                              GDK_GL_MODE_DOUBLE);
   if (!priv->glconfig)
     {
       printf ("Could not setup GL-context!\n");
